### DISCUSSION TITLE
/software-systems-engineering/the-center-of-an-application-ef-and-databases/

### DISCUSSION THREAD

**Discussion by giscus** [thenewobjective/thenewobjective.github.io#158] [Announcements]
# The Center of an Application: EF and Databases

The personal website of Michael L Haufe

https://thenewobjective.com/software-systems-engineering/the-center-of-an-application-ef-and-databases

**Comment by mlhaufe** [comment]
This article has been posted on LinkedIn:
https://www.linkedin.com/pulse/center-application-ef-databases-michael-haufe-v1ale
